docs/providers/ai21.md

```diff
@@ -9,32 +11,44 @@ AI21_API_KEY=
 {% endcode %}
 
 ```typescript
-import { TokenJS, ChatCompletionMessageParam } from 'token.js'
+import { TokenJS } from 'token.js'
 
-// Import and create the token.js client
+// Create the Token.js client
 const tokenjs = new TokenJS()
 
-// Specify OpenAI compatible messages
-const messages: ChatCompletionMessageParam = [{
-  role: 'user',
-  content: `How are you?`,
-}]
-
-// Call the create function
-const result = await tokenjs.chat.completions.create({
-  // Specify the target model and provider
-  provider: 'ai21',
-  model: 'jamba-instruct',
-  messages,
-})
+async function main() {
+  // Create a model response
+  const completion = await tokenjs.chat.completions.create({
+    // Specify the provider and model
+    provider: 'ai21',
+    model: 'jamba-instruct',
+    // Define your message
+    messages: [
+      {
+        role: 'user',
+        content: 'Hello!',
+      },
+    ],
+  })
+  console.log(completion.choices[0])
+}
+main()
```

Comment:
The async main function is unnecessary boilerplate that distracts from the primary content. Other packages don't do it, and I don't think it would benefit us much either. 

**Examples** 
- [nanoid](https://www.npmjs.com/package/nanoid)
- [openai](https://www.npmjs.com/package/openai)
- [jwt-decode](https://www.npmjs.com/package/jwt-decode)

Reply:
Two of the examples you linked don't seem to include async functions, so I don't think they're relevant, and the OpenAI page you linked actually includes async main functions in their usage examples. OpenAI also includes these functions in their API reference. I personally like being able to copy and paste examples and run them as is, so unless you're opinionated on this, I think we should leave them in

Reply:
We could potentially make the async calls on the top level of the file like [Alchemy's API reference](https://docs.alchemy.com/reference/eth-call-polygon-zkevm), since that seems to have slightly less boilerplate. I don't have an opinion between this and defining a `main` function